_health-care/_js/utils/validations.js

```diff
@@ -15,7 +15,34 @@ function isNotBlank(value) {
 }
 
 function isValidSSN(value) {
+  // Conditions:
```

Comment:
Also, cite the source for the validation rules. Like "Taken from the original 1010ez pdf app" or something.